frontend/www/js/omegaup/components/problem/Settings.vue

```diff
@@ -45,7 +47,7 @@
         <input
           name="validator_time_limit"
           v-bind:value="validatorTimeLimit"
-          v-bind:disabled="languages === ''"
+          v-bind:disabled="languages === '' || validator !== 'custom'"
```

Comment:
nit:

```suggestion
          v-bind:disabled="validator !== 'custom'"
```

creo que el hecho de que no haya lenguajes debería implicar que no haya validador, pero dejemos eso para después.

Reply:
En específico, ese campo se debe deshabilitar en cualquiera de  los dos casos:

- Cuando sea lectura sin envíos
- Cuando el validador es distinto al personalizado.

Reply:
sí, pero a lo que me refiero es que si hay una lectura sin envíos, el validador _no debe_ poder ser personalizado, así que sólo hay un caso: si el validador es distinto al personalizado.

pero démosle merge así como está y lo arreglamos luego.

Reply:
Ya entiendo. En otro cambio podemos arreglar eso.